src/installer/utils.py

```diff
@@ -129,7 +129,10 @@ def get_launcher_kind():  # pragma: no cover
 
 def fix_shebang(stream, interpreter):
     # type: (BinaryIO, str) -> BinaryIO
-    """Replace ^#!python shebang in a stream with the correct interpreter."""
+    """Replace ^#!python shebang in a stream with the correct interpreter.
+
+    The original stream should be closed by the caller.
```

Comment:
And the returned stream?

Reply:
Also.

Reply:
See comment above. The returned stream now does not need to be closed, only the original one does.